### DISCUSSION TITLE
Admin-UI v6 can not change password

### DISCUSSION THREAD

**Discussion by nguyennam38** [ServiceStack/Discuss#31] [Q&A] [answered]
I'm using servicestack v6. I tried to change user password by Admin-UI page but user's password can not be changed. I read script of Admin-ui page and see changePassword function don't have new password parameter, 
![change pass](https://user-images.githubusercontent.com/12172654/188255791-48f80f03-b528-43a4-b89d-2ef31aebc6d9.JPG)

changePassword() {
                    this.send(new AdminUpdateUser({ id:this.id }), respose => {
                        this.model.password = ''
                        this.success = 'Password was changed'
                        this.bind(respose)
                    })
                },
That is why password is not changed. Please support me to work the function. thank y

**Comment by mythz** [comment, marked as answer]
This should now be resolved from [this commit](https://github.com/ServiceStack/ServiceStack/commit/3f0378810985d3a3fa9ed9c5127d188fe1ad4950).

This change is available from v6.3.1 that's now [available on MyGet](https://docs.servicestack.net/myget).

thx for reporting 👍
